setup.py

```diff
@@ -42,17 +43,21 @@
         'tests': [
             'coveralls',
             'doubles',
-            'flake8<3',  # see https://github.com/zheller/flake8-quotes/issues/29
+            'flake8',
             'flake8-quotes',
-            'mock<1.1.0',
-            'psycopg2>=2.4.0',
+            'mock',
+            'psycopg2-binary',
             'sqlalchemy>=1.2.0',
-            'pytest>=3.0.0',
+
+            # pytest-tornado isn't compatible with pytest>=4.0.0,
+            # see https://github.com/eugeniy/pytest-tornado/pull/38
+            'pytest>=3.0.0,<4.0.0',
+
             'pytest-cov',
             'pytest-localserver',
             'pytest-mock',
             'pytest-tornado',
-            'basictracer>=2.1,<2.2',
+            'basictracer<3.0.0',
```

Comment:
please change to >=2.1,<3

Reply:
It's done.

I'll appreciate if you share some thoughts about why we have to pin a minimal version here. Thanks!